### PR TITLE
Update historic imagery pricing

### LINKED ISSUE #167: Updated RDC Pricing and Turn Around Time (March 2)
@christoball @mitchellryant  @csidenblad 

Hey Chris, effective March 2nd the public pricing listed in the "Historic Imagery Pricing" section on TNRIS.org will update.  Can you please add the following updates?  Format can look exactly the same as it is currently.

Research:
$60/ Full Historical Search (multiple years or sites)
$30/Single Year Search (one site and one year)
$15/Frame Search (customer must be able to identify specific frames needed)

Imagery:
$10/frame
$20/georeferenced frame (all frames will be georeferenced to WGS 1984)

Delivery Options:
$7 digital download delivery (0-10gb)

Bulk Collection Delivery Option

Customers have the option of ordering any collection that has already **been scanned in full** for the data size price (see Data Reproduction Costs), rather than paying by frame. 

Delivery Policy:

TNRIS requests 3-10 business days for data requests and product turn around.  We do not accommodate any expedited orders.

### PR DESCRIPTION
**Pull request by christoball**
closes #167

### CODE REVIEW

File: templates/rdc/rdc-sub.njk [hunk 1]
```diff
@@ -51,33 +50,30 @@
...
-<div class="bs-callout bs-callout-danger">
-  <p><strong>No Refund Policy:</strong> TNRIS does not offer refunds for services rendered. All data is available through the Public Domain at no cost. All applicable fees are associated with the cost of data reproduction.</p>
-</div>
-<hr>
+        <a class="btn btn-lg btn-tnris btn-block" href="https://data.tnris.org/catalog/%7B%22filters%22%3A%7B%22category%22%3A[%22Historic_Imagery%22]%7D%7D"><img style="width: 20px; margin-bottom: 0 !important;" src="https://tnris-org-static.s3.amazonaws.com/images/baseline_view_comfy_white_36dp.png"> Launch DataHub</a>
+        <hr>
+        <h3>Historical Imagery Pricing</h3>
+        <h4>Research</h4>
+        <ul class="list-nospace">
+          <li>$60/ Full Historical Search (multiple years or sites)</li>
```
Comment (JasonKleinert) [inline templates/rdc/rdc-sub.njk]: Looks like you have an extra space here after "$60/"

File: templates/rdc/rdc-sub.njk [hunk 2]
```diff
@@ -51,33 +50,30 @@
...
+          <li>$30/Single Year Search (one site and one year)</li>
+          <li>$15/Frame Search (customer must be able to identify specific frames needed)</li>
+        </ul>
+        <h4>Imagery Reproduction Costs</h4>
+        <ul class="list-nospace">
+          <li>$10/frame</li>
+          <li>$20/georeferenced frame (all frames will be georeferenced to WGS 1984)</li>
+        </ul>
+        <h4>Delivery Options</h4>
+        <p>$7 digital download delivery (0-10gb)</p>
```
Comment (JasonKleinert) [inline templates/rdc/rdc-sub.njk]: Did you want a "/" after "$7" like you have on the other lines?

**Review by JasonKleinert — CHANGES_REQUESTED** [review]
It looks like the formatting needs to be discussed with RDC. They have provided an updated comment to look to on this issue.

**Comment by christoball** [comment]
@JasonKleinert I've worked out the changes with the RDC and they said it looks good to go on their end.

File: content/markdown/research-distribution-center/historical-imagery-archive.md [hunk 3]
```diff
@@ -1,16 +1,56 @@
...
+<h3>Bulk Collection Option</h3>
+<p>Customers have the option of ordering any collection that has already been <strong>scanned in full</strong> for the data size price (see Data Reproduction Costs), rather than paying by frame.</p>
+<ul>
+  <li>Staff time: <strong>$46.00 per TB</strong></li>
+  <li>Server/computer time: <strong>$0.15 per GB</strong></li>
+</ul>
+<h3>Delivery Options and Policy</h3>
+<p>TNRIS requests 3-10 business days for data requests and product turn around. We do not accommodate any expedited orders.</p>
+<ul>
+  <li>Digital download delivery (0-10gb): <strong>$7.00</strong></li>
```
Comment (adambreznicky) [inline content/markdown/research-distribution-center/historical-imagery-archive.md]: please update the (0-10gb) to be the same as the other 2 options: (0-10GB)

**Comment by adambreznicky** [comment]
@christoball  found one inconsistency. when fixed, we can call it and merge/deploy. looks good though!

**Comment by christoball** [comment]
@adambreznicky fixed in the most recent commit

**Pull request merged**